### ISSUE TITLE
Email/ SMS notification

### ISSUE THREAD

**Issue by lsc0498** [corona-school/backend_new#3]
Implement a logic for email and sms notifications. 

- [ ] Based on Mailjet API
- [ ] Distinction between transactional and non-transactional
- [ ] Operations should be logged
- [ ] User as defined in data model should be taken as recipient
- [ ] Check whether user allows for non-transactional mails

**Comment by ayushpandey8439** [comment]
Is there a pre configured instance of the Mailjet API for SMS notifications?

**Comment by lsc0498** [comment]
We have already used it in corona-school/backend if it's this what you mean 🙂

**Comment by ayushpandey8439** [comment]
Yes, But for now, I configured a new API with my email address. For the SMS notifications, there needs to be credit in the account. If there is already an account which you have setup with the credit, I can use that.

**Comment by lsc0498** [comment]
Here are dev API keys for Mailjet 🙂
https://docs.google.com/document/d/1H_9XADUg7qyuVt49AQEdNsZFAQ94w6scw9frqvCWsgE

**Comment by ayushpandey8439** [comment]
What is the sender for the message configured as. I need to pass it as a parameter in the request.

**Comment by lsc0498** [comment]
I think someone just added your @corona-school.de mail after you triggered some alert😉 But in any case noreply@corona-school.de should work.

**Comment by ayushpandey8439** [comment]
I changed the API Keys and forgot that the default sender in my data model was set to my ID. So that caused it. I hope nothing broke. Sorry about the mistake. Also, noreply@corona-school.de didn't work for me. I tried sending a few emails but none was delivered.

**Comment by ayushpandey8439** [comment]
Hey, I have implemented both Email and SMS notifications. I have also checked if they work or not. The setting I used for connecting to the API, I will put them onto the shared document on google drive.
